Bottom: cc399ed712c028dc8c1764a3f68860c514c8390f
Top:    0cb350b32de54ca673dd3831cf721dc6230c504a
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-05 20:48:05 +0100

hid/gtk: Replace gtk_container_border_width with gtk_container_set_border_width

Replaces the old deprecated API.


---

diff --git a/src/hid/gtk/gui-top-window.c b/src/hid/gtk/gui-top-window.c
index 2c76a0a..cf05e4b 100644
--- a/src/hid/gtk/gui-top-window.c
+++ b/src/hid/gtk/gui-top-window.c
@@ -676,7 +676,7 @@ make_cursor_position_labels (GtkWidget * hbox, GHidPort * port)
    */
   frame = gtk_frame_new (NULL);
   gtk_box_pack_end (GTK_BOX (hbox), frame, FALSE, TRUE, 0);
-  gtk_container_border_width (GTK_CONTAINER (frame), 2);
+  gtk_container_set_border_width (GTK_CONTAINER (frame), 2);
   gtk_frame_set_shadow_type (GTK_FRAME (frame), GTK_SHADOW_ETCHED_OUT);
 
   label = gtk_label_new ("");
@@ -690,7 +690,7 @@ make_cursor_position_labels (GtkWidget * hbox, GHidPort * port)
    */
   frame = gtk_frame_new (NULL);
   gtk_box_pack_end (GTK_BOX (hbox), frame, FALSE, TRUE, 0);
-  gtk_container_border_width (GTK_CONTAINER (frame), 2);
+  gtk_container_set_border_width (GTK_CONTAINER (frame), 2);
   gtk_frame_set_shadow_type (GTK_FRAME (frame), GTK_SHADOW_ETCHED_OUT);
   label = gtk_label_new (" __.__  __.__ ");
   gtk_container_add (GTK_CONTAINER (frame), label);
diff --git a/src/hid/gtk/gui-utils.c b/src/hid/gtk/gui-utils.c
index 0ea661b..3be7931 100644
--- a/src/hid/gtk/gui-utils.c
+++ b/src/hid/gtk/gui-utils.c
@@ -534,7 +534,7 @@ ghid_framed_vbox (GtkWidget * box, gchar * label, gint frame_border_width,
   gtk_container_set_border_width (GTK_CONTAINER (frame), frame_border_width);
   gtk_box_pack_start (GTK_BOX (box), frame, frame_expand, frame_expand, 0);
   vbox = gtk_vbox_new (FALSE, vbox_pad);
-  gtk_container_border_width (GTK_CONTAINER (vbox), vbox_border_width);
+  gtk_container_set_border_width (GTK_CONTAINER (vbox), vbox_border_width);
   gtk_container_add (GTK_CONTAINER (frame), vbox);
   return vbox;
 }
@@ -551,7 +551,7 @@ ghid_framed_vbox_end (GtkWidget * box, gchar * label, gint frame_border_width,
   gtk_container_set_border_width (GTK_CONTAINER (frame), frame_border_width);
   gtk_box_pack_end (GTK_BOX (box), frame, frame_expand, frame_expand, 0);
   vbox = gtk_vbox_new (FALSE, vbox_pad);
-  gtk_container_border_width (GTK_CONTAINER (vbox), vbox_border_width);
+  gtk_container_set_border_width (GTK_CONTAINER (vbox), vbox_border_width);
   gtk_container_add (GTK_CONTAINER (frame), vbox);
   return vbox;
 }
